lib/wiki_discouraged_article_manager.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+require_dependency "#{Rails.root}/lib/importers/category_importer"
+
+class WikiDiscouragedArticleManager
+  def retrieve_wiki_edu_discouraged_articles
+    category_title = ENV['blocked_assignment_category']
+    discouraged_articles = CategoryImporter.new(Wiki.default_wiki)
```

Comment:
Instead of calling `CategoryImporter` directly to get the titles, you can just find the `Category` record and then use `#refresh_titles` to do this.